content/docs/nav.yml

```diff
@@ -1,89 +1,89 @@
-- title: Installation
+- title: Instalacija
   items:
     - id: getting-started
-      title: Getting Started
+      title: Prvi koraci
     - id: add-react-to-a-website
-      title: Add React to a Website
+      title: Dodavanje React-a
     - id: create-a-new-react-app
-      title: Create a New React App
+      title: Nova React aplikacija
     - id: cdn-links
-      title: CDN Links
+      title: CDN linkovi
     - id: release-channels
       title: Release Channels
-- title: Main Concepts
+- title: Glavni koncepti
   isOrdered: true
   items:
     - id: hello-world
-      title: Hello World
+      title: Zdravo, svete
     - id: introducing-jsx
-      title: Introducing JSX
+      title: Upoznavanje JSX
     - id: rendering-elements
-      title: Rendering Elements
+      title: Prikazivanje elemenata
     - id: components-and-props
-      title: Components and Props
+      title: Komponente i svojstva
     - id: state-and-lifecycle
-      title: State and Lifecycle
+      title: Stanje i životni vek
     - id: handling-events
-      title: Handling Events
+      title: Rukovanje događajima
     - id: conditional-rendering
-      title: Conditional Rendering
+      title: Uslovno prikazivanje
     - id: lists-and-keys
-      title: Lists and Keys
+      title: Liste i ključevi
     - id: forms
-      title: Forms
+      title: Forme
     - id: lifting-state-up
-      title: Lifting State Up
+      title: Uzdizanje stanja naviše
     - id: composition-vs-inheritance
-      title: Composition vs Inheritance
+      title: Kompozicija vs nasleđivanje
     - id: thinking-in-react
-      title: Thinking In React
-- title: Advanced Guides
+      title: Razmišljanje u React-u
+- title: Napredni vodiči
```

Comment:
It's not `Napredni vodici` but `Napredni vodic`

Reply:
Fixed :+1: